docs/content/preview/benchmark/sysbench-ysql.md

```diff
@@ -128,34 +128,46 @@ You can choose to run the following workloads individually:
 Before starting the workload, load the data as follows:
 
 ```sh
-$ sysbench <workload>                       \
-      --tables=10                           \
-      --table-size=100000                   \
-      --range_key_partitioning=true         \
-      --db-driver=pgsql                     \
-      --pgsql-host=<comma-separated-ips>    \
-      --pgsql-port=5433                     \
-      --pgsql-user=yugabyte                 \
-      --pgsql-db=yugabyte                   \
-      prepare
+sysbench <workload> \
+--pgsql-host=<comma-separated-ips> \
```

Comment:
indented layout was nicer

Reply:
The reason I did this is that in DB audit ybm cli PR, Atharva suggested we keep one space before backslash; his comment - https://github.com/yugabyte/yugabyte-db/pull/25483#discussion_r1904384973

Reply:
one space before backslash is fine; I'm talking about indenting the lines after the first line

Reply:
Okay, got it, thanks!